CONTRIBUTING.md

```diff
@@ -70,16 +70,17 @@ You can join the OSRD team on [transifex](https://explore.transifex.com/osrd/osr
 
 ## Contribute code
 
-[osrd.fr/docs/contribute/code](https://osrd.fr/en/docs/guides/contribute/code).
+To comply with the [DCO](http://developercertificate.org/), all commits must
+include a Signed-off-by line. You can find more information about this here:
+[osrd.fr/docs/guides/contribute/contribute-code/commit-conventions/#the-developer-certificate-of-origin](https://osrd.fr/en/docs/guides/contribute/contribute-code/commit-conventions/#the-developer-certificate-of-origin)
+
+For more advice on how to contribute, follow that link:
+[osrd.fr/docs/contribute/contribute-code](https://osrd.fr/en/docs/guides/contribute/contribute-code).
 
 ### Pull request / code review
 
-[osrd.fr/docs/contribute/code/#share-your-changes](https://osrd.fr/en/docs/guides/contribute/code/#share-your-changes).
+[osrd.fr/docs/contribute/contribute-code/share-your-changes](https://osrd.fr/en/docs/guides/contribute/contribute-code/share-your-changes).
```

Comment:
Note to self, this page is sitting in a PR: https://github.com/OpenRailAssociation/osrd-website/pull/231